html2docx.py

```diff
@@ -38,6 +39,10 @@ def handle_starttag(self, tag: str, attrs: List[Tuple[str, str]]) -> None:
             self.finish()
             level = int(tag[-1])
             self.p = self.doc.add_heading(level=level)
+        elif tag == "a":
+            href = [val for name, val in attrs if name == "href"]
```

Comment:
To get the first value in an iteratable, you can use the expression:

```suggestion
            href = nex((val for name, val in attrs if name == "href"), None)
```

https://docs.python.org/3/library/functions.html#next